answerkey/laziness/14.answer.md

```diff
@@ -6,4 +6,7 @@ def startsWith[A](s: Stream[A]): Boolean =
   zipAll(s).takeWhile(!_._2.isEmpty) forAll {
     case (h,h2) => h == h2
   }
+  
+def startsWithUsingFoldRight[A](s: Stream[A]): Boolean =
```

Comment:
For example, when `Stream(1).startsWith(Stream(1, 2))`, it should return false, but I feel this implementation would return true.

Reply:
True